src/Sign.SignatureProviders.TrustedSigning/RSATrustedSigning.cs

```diff
@@ -15,7 +15,7 @@ internal sealed class RSATrustedSigning : RSA
         private readonly CertificateProfileClient _client;
         private readonly string _accountName;
         private readonly string _certificateProfileName;
-        private readonly X509Certificate2 _certificate;
+        private readonly RSA _rsaPublicKey;
```

Comment:
X509Certificate2 is not specifically only for RSA, does it break anything now?

Reply:
We were doing `_certificate.GetRSAPublicKey()!` in the property I moved that to the constructor to make sure it's disposed after we are done using it. Functionally nothing was changed.